cumulus/client/consensus/relay-chain/src/import_queue.rs

```diff
@@ -55,6 +56,8 @@ where
 		&mut self,
 		mut block_params: BlockImportParams<Block>,
 	) -> Result<BlockImportParams<Block>, String> {
+		block_params.fork_choice = Some(ForkChoiceStrategy::LongestChain);
```

Comment:
We could also set `Custom(true)` here. We had some discussion with @skunert  about this, but we don't have final conclusion on this.

`Custom(true)` would trigger more maintenance in transaction pool, making it up-to-date for all forks, but I don't know if building blocks would be actually synced with that state.

Reply:
I mean this is the relay chain verifier, afaik that is pretty  niche anyway and mostly for historic blocks during sync. I think either is fine here :thinking:

Reply:
Please just bring back the old logic with "block_params.source == InitialSync`.